src/account/operations/transfer/high_level/burn_foundry.rs

```diff
@@ -0,0 +1,203 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashSet;
+
+use iota_client::bee_message::{
+    output::{AliasId, AliasOutputBuilder, FoundryId, Output, OUTPUT_COUNT_MAX},
+    payload::transaction::TransactionId,
+};
+
+use crate::{
+    account::{
+        handle::AccountHandle, operations::transfer::TransferResult, types::OutputData, SyncOptions, TransferOptions,
+    },
+    Error,
+};
+
+impl AccountHandle {
+    /// Function to destroy foundry
+    pub async fn burn_foundry(
```

Comment:
In https://github.com/lzpap/tips/blob/master/tips/TIP-0018/tip-0018.md#additional-transaction-semantic-validation-rules-2 it's called `Destroy`, I think we should also use that for the names of methods that destroy outputs and only for burning of native tokens use `burn`

Reply:
Hm actually for NFTs it says burning https://github.com/lzpap/tips/blob/master/tips/TIP-0018/tip-0018.md#consumed-outputs-2, so not sure what we should use

Reply:
Use destroy here